GUI/src/app/pages/authorization-page/authorization-page.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-authorization-page',
+  templateUrl: './authorization-page.component.html',
+  styleUrls: ['./authorization-page.component.css']
+})
+export class AuthorizationPageComponent {
+  authorizationForm = new FormGroup({
```

Comment:
please, move initialization to constructor + use FormBuilder

Reply:
done